app/views/plantings/show.html.haml

```diff
@@ -67,6 +67,9 @@
     %p= render 'plantings/progress', planting: @planting, show_explanation: true
 
     = render 'plantings/actions', planting: @planting
+    - if !@planting.photos.empty? || (can?(:edit, @planting) && can?(:create, Photo))
+      - if can?(:create, Photo) && can?(:edit, @planting)
+        = link_to "Add photo", new_photo_path(type: "planting", id: @planting.id), class: 'btn btn-primary btn-xs'
```

Comment:
I see we're already testing for the presence of an "Add photo" button on this page (in `spec/views/plantings/show.html.haml`): @Br3nda, do you think we need a check for this additional button?

Reply:
It's a bit confusing to read like that. Can it be any clearer?